module-apps/application-messages/include/application-messages/ApplicationMessages.hpp

```diff
@@ -69,6 +69,8 @@ namespace app
 
       private:
         ActionResult handleCreateSmsAction(std::unique_ptr<gui::SwitchData> data);
+
+        bool phoneIsLocked = false;
```

Comment:
Maybe it will look better if the states will be introduced for example enum LOCKED, UNLOCKED instead of keeping the state in the bool variable

Reply:
I think so, too

Reply:
done.